lib/config/env.js

```diff
@@ -39,5 +39,36 @@ function getConfig(env) {
     }
   });
 
+  if (env.GITHUB_COM_TOKEN) {
+    config.endpoints.push({
+      platform: 'github',
+      endpoint: 'github.com',
+      token: env.GITHUB_TOKEN,
+    });
+  }
+  if (env.GITHUB_TOKEN) {
+    config.endpoints.push({
+      platform: 'github',
+      endpoint: env.GITHUB_ENDPOINT,
```

Comment:
So this will be `undefined` at times when `GITHUB_ENDPOINT` is not set, and that's ok? Or should we add the default value here to make it more obvious? Same applies to GitLab